docs/research/MO-gcylc-rc-report.md

```diff
@@ -0,0 +1,117 @@
+# Usage Of `gcylc.rc` At The MetOffice
+
+## Overview
+
+* Users:
+  * ~700 users in total.
+  * ~150-200 simultaneous users at any one time.
+  * 76 users have gcylc.rc files.
+  * There isn't much customisation of `gcylc` at present.
+  * A lot of people are hard-coding the default settings.
+
+* Views:
+  * The most changed setting is `initial views` people like to customise this.
+  * People who work with large suites often disable the graph setting.
+  * There isn't much love for the `dot` view.
+
+* Visual impairment:
+  * Two users increased the size of the task icons.
+  * Three users choose the `high-contrast` theme.
+  * Four users chose the `color-blind` theme.
+  * There are 6 custom task/job state colour themes these appear to be driven by
+    preference rather than visibility issues.
+
+## Results
+
+Here are the values users have set for the `gcylc.rc` settings along with
+the number of times that value was used:
+
+
+### `dot icon size`
+* 6 `medium`
+* 5 `small`
+* 2 `large`
+
+### `initial side-by-side views`
+* 9 `True`
+* 6 `False`
+* 1 `true`
+
+### `initial views`
+
+* 17 `text, graph`
+* 12 `text`
+* 10 `graph, text`
+* 7 `dot`
+* 6 `text, dot`
+* 5 `graph`
+* 2 `graph,text`
+* 1 `text,graph`
+* 1 `dot, text`
+* 1 `dot, none`
+
+That boils down to:
+
+* 48 `text`
+* 34 `graph`
+* 15 `dot`
+* 1 `none`
+
+### `maximum update interval`
+
+### `sort by definition order`
+* 3 `True`
+* 1 `false`
+
+### `sort column`
+* 3 `state`
+* 1 `none`
+
+### `sort column ascending`
+* 1 `True`
+
+### `sub-graphs on`
+
+### `task filter highlight color`
+* 2 `PowderBlue`
+
+### `task states to filter out`
+* 1 `waiting`
+* 1 `succeeded`
+* 1 `runahead`
+* 1 `held`
+
+### `transpose dot`
+* 1 `True`
+* 1 `False`
+
+### `transpose graph`
+* 3 `True`
+* 1 `true`
+* 1 `False`
+
+### `ungrouped views`
+* 20 `graph`
+* 10 `text`
+* 10 `dot`
+
+### `use theme`
+* 7 `default`
+* 5 `solid`
+* 3 `high-contrast`
+* 3 `color-blind`
+* 2 `SMS`
+* 2 `PinkRun`
+* 1 `multi`
+* 1 `custom`
+* 1 `adam`
```

Comment:
Maybe we could group the non-standard themes (set by only user each) into some umbrella category such as 'user's custom individual theme'? Clearly Adam & Anette (& some other users not naming their themes after themselves) had their own visualisation preferences :smile:!

Reply:
Cylc provides:
* 7 `default`
* 5 `solid`
* 3 `high-contrast`
* 3 `color-blind`

The rest are custom.

Reply:
The point I was trying to make (but didn't convey at all, sorry) is that it doesn't matter what those themes are called, what is important is that each user has defined their own. So a group to collate all users defining their own theme (i.e. the sum of the totals for all themes not on your list above) would seem more useful to me.

Saying that, there appear to be some user-defined themes e.g. ``PinkRun`` that multiple users have applied. Any idea how that has happened? Can users share themes? (I doubt it is just a coincidental choice of the same name).

Reply:
https://github.com/cylc/cylc-flow/blob/7.8.3/etc/gcylc.rc.eg#L19

Reply:
Oh dear :joy: Perhaps those poor souls forgot they tried that out & think the default theme uses magenta?